docs/guidelines/platforms/ios/guideline_understandable_ios.md

```diff
@@ -177,6 +228,91 @@ usernameHolderView.shouldGroupAccessibilityChildren = true
 
 A user enters some data into a user interface element without knowing what the end result is and/or triggering an unannounced change of context in the process.
 
+### Error Suggestion (WCAG 3.3.3 - Level AA)
+
+To provide the best user experience, the application should suggest solutions for input errors when they are detected.
+
+> This technique covers point *3.3.3 Error Suggestion - Level AA of the WCAG standard.*
+
+#### ✅ Success technique(s)
+
+When an error is detected, the application should provide a suggestion for the user on how to correct it. This can be done by providing a hint or a description of the error, or providing a suggestion on how to correct it.
+
+This is not just an accessibility feature, but also a usability feature, as it helps users to understand what they did wrong and how to correct it. Of course, if the suggestion is presented, this should be done in accessible way (e.g. using accessibility label).
+
+*Note: This should be avoided if the error suggestion would jeopardize the security or purpose of the content.*
+
+### Error Prevention (Legal, Financial, Data) (WCAG 3.3.4 - Level AA)
+
+To prevent users from making mistakes, the application should provide a mechanism before finalizing a transaction that involves legal, financial, or data-sensitive actions.
+
+> This technique covers point *3.3.4 Error Prevention (Legal, Financial, Data) - Level AA of the WCAG standard.*
+
+#### ✅ Success technique(s)
+
+To satisfy this criterion, the app should provide at least one of the following mechanisms:
+
+- **Reversible**: The submission is reversible.
+- **Checked**: Data entered by the user is checked for input errors and the user is provided with an opportunity to correct them.
+- **Confirmed**: A mechanism is available for reviewing, confirming, and correcting information before finalizing the submission.
+
+Based on the type of the application, the mechanism should be implemented in a way that is most suitable for the user and the application's purpose.
+
+### Redundant Entry (WCAG 3.3.7 - Level A)
+
+Ensure that multi-step processes are easy for users by not requesting the same information multiple times in a session, as this can be challenging for those with cognitive disabilities. This approach enhances accessibility by reducing memory load and simplifying tasks.
```

Comment:
```suggestion
Ensure that multi-step processes are user-friendly by not requesting the same information multiple times in a session, as this can be challenging for those with cognitive disabilities. This approach enhances accessibility by reducing memory load and simplifying tasks.
```